Index: CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>cmake_minimum_required(VERSION 3.10)\n\nproject(TerrainGeneration VERSION 1.0)\n\nadd_compile_options(-g -Wall -Wextra)\n\nset(CMAKE_CXX_STANDARD 17)\nset(CMAKE_CXX_STANDARD_REQUIRED True)\nset(CMAKE_EXPORT_COMPILE_COMMANDS True)\n\nadd_executable(TerrainGeneration\n    src/main.cpp\n    src/terrain.cpp\n    src/perlin.cpp\n    src/shader.cpp\n    src/utility.cpp\n    src/vec2.cpp\n    src/camera.cpp\n    #src/terrain_renderer.cpp\n    src/chunk.cpp\n    external/glad/glad.c\n)\n\nfind_package(glfw3 REQUIRED)\n\ntarget_include_directories(TerrainGeneration PRIVATE\n    ${PROJECT_SOURCE_DIR}/include\n    ${PROJECT_SOURCE_DIR}/external/glad\n)\n\ntarget_link_libraries(TerrainGeneration PRIVATE glfw)\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	(revision 76f5930c41ef722bf237b8f4d4ba66c4ac9458dc)
+++ b/CMakeLists.txt	(date 1721195001779)
@@ -16,7 +16,7 @@
     src/utility.cpp
     src/vec2.cpp
     src/camera.cpp
-    #src/terrain_renderer.cpp
+    src/terrain_renderer.cpp
     src/chunk.cpp
     external/glad/glad.c
 )
